extensions/amp-recaptcha-input/amp-recaptcha-input.md

```diff
@@ -78,6 +78,10 @@ grecaptcha.execute('reCAPTCHA_site_key', {action: 'reCAPTCHA_example_action'});
     <td width="40%"><strong>data-action (required)</strong></td>
     <td><a href="https://developers.google.com/recaptcha/docs/v3">reCAPTCHA v3</a> action to be executed on form submission.</td>
   </tr>
+  <tr>
+    <td width="40%"><strong>global [optional]</strong></td>
```

Comment:
It looks like the other recaptcha-forwarded attributes are prefixed with `data-`, let's use this so the API is self-consistent.

Reply:
Updated to use `data-`, thanks!